### DISCUSSION TITLE
Memory Map für CPM-65 auf dem JC II

### DISCUSSION THREAD

**Discussion by Dietrich-L** [Dietrich-L/CPM-65_for_JUNIOR_COMPUTER_II#1] [General]
Ein Vorschlag für eine Memory map liegt in DOCS.

Fragen:
- kann ich den ganzen Bereich $2000 - $DFFF benutzen?
- Passen die Zero page Adressen?

**Comment by CC2ee** [comment]
Der Bereich ist dann frei. Die Zero Page Adressen sollten sich von meiner Seite nicht mehr ändern. Theoretisch kannst du natürlich auch die Bereiche nehmen, die ich für ehBasic ausgeklammer habe.

**Reply by Dietrich-L** [reply]
Ok. Sollte passen
